tests/test_recipient_validation.py

```diff
@@ -88,6 +86,8 @@
     'email+middle"-quotes@domain.com',
     '"quoted-local-part"@domain.com',
     '"quoted@domain.com"',
+    'lots-of-dots@domain..gov..uk',
```

Comment:
Can you add a test case for `lots..of..dots@domain.com` with a comment to say that it’s technically valid but that we’re not allowing it?